installer/roles/local_docker/tasks/compose.yml

```diff
@@ -12,3 +12,6 @@
 - name: Start the containers
   docker_service:
     project_src: "{{ docker_compose_dir }}"
+    
+- name: Update CA trust in awx_web container
+  command: docker exec awx_web_1 '/usr/bin/update-ca-trust'
```

Comment:
In the standalone installer, we only run this command if the service starting task has a `changed` status to keep things idempotent. I think doing a similar check here would also be a good idea.

Reply:
I see what you mean, I added status check.